cmd/device/add/add.go

```diff
@@ -113,9 +114,9 @@ func processFile(fname string) {
 		dev.Origin = millis
 		id, err := addDevice(dev)
 		if err != nil {
-			fmt.Println("......Error: ", err.Error())
+			fmt.Println("Error: ", err.Error())
```

Comment:
Once we have parsed a toml file .. is the error that is possible stemming from issues adding the device to edgeX?

Reply:
yes ..